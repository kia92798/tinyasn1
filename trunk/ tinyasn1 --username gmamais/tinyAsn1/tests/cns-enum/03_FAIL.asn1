MY-MODULE DEFINITIONS AUTOMATIC TAGS::= BEGIN
	
	myIntVal INTEGER ::= 4
	
	MyEnum ::= ENUMERATED {red, blue(0), green(myIntVal)}
	
	myEnumVar  MyEnum ::= red
	
	myEnumVar2  MyEnum ::= myEnumVar

	JustRedBlue ::= MyEnum(red .. blue)


	
END
