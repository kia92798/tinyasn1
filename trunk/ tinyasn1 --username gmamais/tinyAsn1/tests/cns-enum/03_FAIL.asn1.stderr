Error2 in Line:11, col:24 . This type of constraint 'VALUE_RANGE_EXPR' cannot appear under ENUMERATED
